ca/ca.go

```diff
@@ -415,7 +414,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 	serialBytes[0] = byte(ca.prefix)
 	_, err = rand.Read(serialBytes[1:])
 	if err != nil {
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("boulder/ca.IssueCertificate: failed to generate serial: %s", err)
```

Comment:
I think it might be worth making a `const` for the `ca.IssueCertificate:` err prefix. Otherwise it's a number of places to update if the function is renamed or parts of this logic extracted. Thoughts?